components/api/src/main/java/com/hotels/styx/api/extension/service/BackendService.java

```diff
@@ -44,6 +44,7 @@
  */
 public final class BackendService implements Identifiable {
     public static final int DEFAULT_RESPONSE_TIMEOUT_MILLIS = 1000;
+    public static final int USE_DEFAULT_MAX_HEADER_SIZE = 0;
```

Comment:
Why do we set the default max header size to zero? Should it be something like 16KB etc?

Reply:
The default max header size is 8K in HttpConfig. 0 means use the default value...